backend/app/api/v1/config.py

```
@@ -72,3 +102,37 @@ def delete_config(config_definition_key: str, config_key: str):
+        The response for the list configurations request.
+    """
+    try:
+        configs = l_config(
+            config_definition_key,
+            page,
+            limit,
+        )
+
+    except Exception as e:
+        logger.exception(f"Error listing configurations: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)